crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -560,10 +560,11 @@ where
         &self,
         payload: ExecutionPayloadV1,
     ) -> EngineApiResult<PayloadStatus> {
-        let payload = ExecutionPayload::from(payload);
+        let payload =
+            ExecutionData { payload: payload.into(), sidecar: ExecutionPayloadSidecar::none() };
```

Comment:
next we can change these functions so that they take the execcutionpayload as arg